subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -486,7 +486,7 @@ void lll_isr_cleanup(void *param)
 	radio_tmr_stop();
 
 	err = lll_hfclock_off();
-	LL_ASSERT(!err || err == -EBUSY);
+	LL_ASSERT(err >= 0);
```

Comment:
Why accept positive error values?

Reply:
it is using onoff service underneath which returns non-negative for success: https://github.com/zephyrproject-rtos/zephyr/blob/979e124e21fb05cfedf8ed12078983d40e2f0b6c/include/sys/onoff.h#L338